pages/index.tsx

```diff
@@ -32,40 +41,43 @@ const Home: NextPage = () => {
 			renderCell: (params: GridRenderCellParams) => (
 				<Link
 					href={{
-						pathname: `/class/${params.row.course_id}`,
-						query: { classid: params.row.course_id, title: params.row.name },
+						pathname: `/class/${params.row.courseId}`,
+						query: { courseId: params.row.courseId, title: params.row.name },
 					}}
 				>
 					{params.row.name}
 				</Link>
 			),
 		},
-		{ field: 'course_id', headerName: 'Course ID', flex: 1 },
-		{ field: 'aliases', headerName: 'Aliases', flex: 1, hide: true },
+		{ field: 'courseId', headerName: 'Course ID', flex: 0.5 },
+		{ field: 'avgDifficulty', headerName: 'Difficulty (out of 5)', flex: 0.5, valueGetter:(params:any)=>Math.round(params.row.avgDifficulty * 10) / 10 },
+		{ field: 'avgWorkload', headerName: 'Workload (hrs/wk)', flex: 0.5, valueGetter:(params:any)=>Math.round(params.row.avgWorkload * 10) / 10 },
+		{ field: 'avgOverall', headerName: 'Overall (hrs/wk)', flex: 0.5 , valueGetter:(params:any)=>Math.round(params.row.avgOverall * 10) / 10},
+		{ field: 'numReviews', headerName: 'Number of Reviews', flex: 0.5 },
+		{ field: 'isDeprecated', headerName: 'is Deprecated?', flex: 0,hide:true},
+		{ field: 'aliases', headerName: 'Aliases', flex: 0,hide:true},
 	]
 	const [loading, setLoading] = useState<boolean>()
 	const [classes, setClasses] = useState<Array<ClassData>>([])
 
 	useEffect(() => {
 		setLoading(true)
-		// fetch('https://omshub-readonly.gigalixirapp.com/classes')
-		fetch('https://omshub-api.gigalixirapp.com/api/classes')
-			.then((res) => res.json())
-			.then((classes) => {
-				//Clean data
-				classes = classes.map((data: object, index: number) => ({
-					...data,
-					id: index,
-				}))
-				setClasses(classes)
-				setLoading(false)
-			})
-			.catch((err) => {
-				setLoading(false)
-				console.log(err)
-			})
+		
+		getCourses().then((courses)=>{
+			const classes=courses.map((data:ClassData,index:number)=>({
```

Comment:
`getCourses()` is now returning an object FYI (i.e., `{ `CS-6300`: { ... }, ... }`, so we'll have to adjust these types of "backend calls" accordingly. Would probably help to create a utility function in `src` to convert an object of objects to an array of objects...

Reply:
Update: FYI in most recent `main` (as of this writing), I created such a function in `/src/utilities` (cf. `mapToArray()`)